### DISCUSSION TITLE
Request to Add an option to change the title in tools bar

### DISCUSSION THREAD

**Discussion by azizly** [amitmerchant1990/notepad#40] [Ideas]
Hi there, great app still using it since last year, my team mates also using it 
it would be helpful if there is an option to change the title in the tools bar, just a text box, where i can click and put the titel before hitting Ctrl + S 

Thanks

**Comment by amitmerchant1990** [comment]
Can you please elaborate more?

Do you want to save the notes with a different name or change the website title based on your needs?
